development.ini

```diff
@@ -15,7 +15,7 @@ jujugui.combine = true
 jujugui.gzip = true
 jujugui.cachebuster = dev
 jujugui.socketTemplate = /environment/$uuid/api
-
+jujugui.gisf = false
```

Comment:
Might be worth adding a comment where this variable is used so we know what it means or maybe even rename it to something more self explanatory :).